<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>256</width>
     <height>88</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btn1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn7">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QPushButton" name="btn3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QPushButton" name="btn9">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>btn1</zorder>
  <zorder>btn2</zorder>
  <zorder>btn3</zorder>
  <zorder>btn4</zorder>
  <zorder>btn5</zorder>
  <zorder>btn6</zorder>
  <zorder>btn7</zorder>
  <zorder>btn8</zorder>
  <zorder>btn9</zorder>
  <zorder>btn3</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
